TESTING.asciidoc

```diff
@@ -364,9 +376,34 @@ quality boxes available in vagrant atlas:
 
 * sles-11
 
-We're missing the following because our tests are very linux/bash centric:
+=== Testing packaging on Windows
+
+The packaging tests also support Windows Server 2012R2 and Windows Server 2016.
+Unfortunately we're not able to provide boxes for them in open source use
+because of licensing issues. Any Virtualbox image that has WinRM and Powershell
+enabled for remote users should work.
+
+Testing on Windows requires the https://github.com/criteo/vagrant-winrm[vagrant-winrm] plugin.
+
+------------------------------------
+vagrant plugin install vagrant-winrm
+------------------------------------
+
+Specify the image IDs of the Windows boxes to gradle with the following environment
+variables. These are used by both the gradle build and Vagrant, so if they're exported
+the Windows boxes will be recognized when you run gradle commands, or Vagrant commands
+directly.
 
-* Windows Server 2012
+* `VAGRANT_WINDOWS_2012R2_BOX`
```

Comment:
I think gradle properties might be better to use here. That way we could set them in `.gradle/gradle.properties`, or configure them via another gradle project within the multi project build in elasticsearch-extra. These could be `vagrant.windows-2012r2.id` and `vagrant.windows-2016.id`. They can still be passed into vagrant via env vars, but from the gradle perspective I think this would be better.

Reply:
Note that to allow them to be set by a sibling project, we need to load them lazily (execution time).